static/js/messages.js

```diff
@@ -30,7 +30,6 @@ function handleMessage(msg) {
                     });
                 });
                 showApps(api);
-                chrome.app.window.current().restore();
```

Comment:
Why was this line removed?  With this line removed, after exiting a stream my window is always fullscreen.  With this line in place, the previous dimensions of the window are restored